packages/api-reference/src/components/Content/Content.vue

```diff
@@ -127,10 +127,14 @@ const introCardsSlot = computed(() =>
 }
 .introduction-card-item {
   padding: 9px;
-  border-bottom: var(--scalar-border-width) solid var(--scalar-border-color);
   display: flex;
   flex-direction: column;
-  justify-content: center;
+  justify-content: start;
+}
+@media (max-width: 900px) {
```

Comment:
I think this uses a container query elsewhere, @cameronrohani?

See [here](https://github.com/scalar/scalar/blob/main/packages/api-reference/src/components/ClassicHeader.vue#L24) or [here](https://github.com/scalar/scalar/blob/main/packages/api-reference/src/components/Content/Content.vue#L164) and the [container](https://github.com/scalar/scalar/blob/main/packages/api-reference/src/components/Content/Content.vue#L108-L111).